.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+end_of_line = lf
```

Comment:
What OS are you using where this was causing an issue? Just curious. Or is this a Vim setting?

Reply:
I didn't hit it, but as an OSS project, good to protect against any potential future windows users.

Reply:
The indent style and indent size settings I totally did use, though.  Apparently I need to get full kotlin support going in my vim instance.  This is just a shortcut for the most common settings.